src/pages/EmployeeProfile/components/BannerItem.jsx

```diff
@@ -0,0 +1,24 @@
+//import
+import { Avatar, Paper, Stack } from "@mui/material";
+import "./style.css"
+
+import Image from '../media/plant-banner.jpg'
+
+export default function BannerItem() {
+
+    const styles = {
+        paperContainer: {
+            backgroundImage: `url(${Image})`
+        }
+    }
```

Comment:
You can use the styles directly in the component using the sx Tag

Reply:
![image](https://github.com/SE2-WS23/Skillverse-Web/assets/65715000/84ba43de-4147-460e-a6b1-81cbcac0e7d6)